csharp/src/SeedLang/Engine.cs

```diff
@@ -34,8 +34,8 @@ public class Engine {
     private readonly RunMode _runMode;
     private readonly VM _vm = new VM();
 
-    // The AST tree of the source code.
-    private AstNode _astNode;
+    // The AST tree of the program.
+    private Statement _program;
```

Comment:
Why don't name this variable "_astTree" or "_programAsAst" ?

Reply:
Fixed.